docs/reference/requirement-sets/identity-api-requirement-sets.md

```diff
@@ -14,7 +14,7 @@ Office Add-ins run across multiple versions of Office. The following table lists
 
 |  Requirement set  | Office 2013 or later on Windows<br>(one-time purchase) | Office on Windows<br>(connected to Office 365 subscription) |  Office on iPad<br>(connected to Office 365 subscription)  |  Office on Mac<br>(connected to Office 365 subscription)  | Office on the web  | SharePoint Online | OneDrive.com |Outlook.com & Exchange Online|
 |:-----|-----|:-----|:-----|:-----|:-----|:-----|:-----|:-----|
-| IdentityAPI 1.1  | N/A | Preview<b>*</b> | Coming soon | Preview<b>*</b> | Preview<b>*</b> | Preview<b>*</b>| Coming soon | Coming soon |
```

Comment:
So there's not going to be a 1.1?

Reply:
@AlexJerabek There was a 1.1 for 2.5 years, but it is ambiguous whether it was GAed or not. For now, the decision (as of this afternoon) is to write as though 1.2 is the only version there ever was.